www/css/dark.css

```diff
@@ -1,7 +1,8 @@
 body {
```

Comment:
The new footer needs to be styled

![image](https://user-images.githubusercontent.com/32238504/62171320-3aa55e00-b337-11e9-868b-b88ab780e003.png)